src/sentry/integrations/slack/views/link_team.py

```diff
@@ -118,25 +120,27 @@ def handle(self, request: Request, signed_params: str) -> HttpResponse:
             )
 
         if not form.is_valid():
-            return render_error_page(request, body_text="HTTP 400: Bad request")
+            return render_error_page(request, status=400, body_text="HTTP 400: Bad request")
 
         team_id = int(form.cleaned_data["team"])
         team = teams_by_id.get(team_id)
         if not team:
-            return render_error_page(request, body_text="HTTP 404: Team does not exist")
+            return render_error_page(request, status=404, body_text="HTTP 404: Team does not exist")
```

Comment:
This also happens if team exists and user doesn't have sufficient permission to it. Do you want to edit the message to reflect that?

Reply:
Changed to `HTTP 404: Team does not exist or you do not have sufficient permission to link a team`